test/membrane_http_adaptive_stream/integration_test/source_test.exs

```diff
@@ -157,11 +189,18 @@ defmodule Membrane.HTTPAdaptiveStream.Source.Test do
     Testing.Pipeline.terminate(pipeline)
   end
 
-  defp hls_to_file_pipeline_spec(url, audio_transcoder, audio_result_file, video_result_file) do
+  defp hls_to_file_pipeline_spec(
+         url,
+         audio_transcoder,
+         audio_result_file,
+         video_result_file,
+         start_at \\ 0
```

Comment:
If you want to use zero time anywhere, please use `Membrane.Time.seconds(0)` or something similar. Fact, that `0` equals `Membrane.Time.seconds(0)` is a part of private API of `membrane_core` and we shouldn't rely anywhere on it. It also makes the code harder to understand